tuxemon/db.py

```diff
@@ -1019,6 +1033,21 @@ def db_entry(self, table: TableName, slug: str) -> bool:
             return True
         return False
 
+    def multiple(self, num: int, check_with: int) -> bool:
```

Comment:
are you sure we need a function that just does `X % Y == 0`?

Reply:
I'm going to remove it, I used it for maintaining order